### ISSUE TITLE
README OnViewport OnIntersect

### ISSUE THREAD

**Issue by ElliotPsyIT** [yoshuawuyts/on-intersect#1]
@yoshuawuyts First, thanks so much for Choo and the collection of libraries and micro-libraries you've been building to support web development!  Coming from Angular, and through you're work (and support from others) I'm having a blast learning the lower level functionality to build apps that are much more performant with much less overhead.  

Minor issue on the README.  As you see, the sample code defines an `onViewport` function (as done in the source), but uses a onIntersect function.  Obviously just a typo in the README example.  FWIW, I'd prefer - and assume you do - the use of `onIntersect` in application code as shown in the example.js in the repo. 

```
var onViewport = require('on-intersect')

...

if (intersectExists()) {
  var stopObserving = onIntersect(el, onEnter, onExit)
}
```

**Comment by yoshuawuyts** [comment]
Oh nice one, PR welcome!

**Comment by yoshuawuyts** [comment]
Fixed in `v2.0.0`! :tada:
